docs/abc.rst

```diff
@@ -250,3 +250,19 @@ experimental APIs without issue.
         .. versionchanged:: 3.0
 
             Added *jwt* and *oauth_token*.
+
+    .. coroutine:: graphql(query, *, endpoint, **variables)
```

Comment:
Looks like this is missing the default `endpoint="https://api.github.com/graphql"`

Reply:
It's now fixed. Thanks for the catch (which happens to be for a feature I included specifically for you 😄 )!

Reply:
Thanks! :laughing: